packages/builder/src/components/start/CreateAppModal.svelte

```diff
@@ -26,7 +26,15 @@
 
   const values = writable({ name: "", url: null })
   const validation = createValidationStore()
-  $: validation.check($values)
+
+  $: {
+    const { name, url } = $values
+
+    validation.check({
+      name,
+      url: url?.[0] === "/" ? url.substring(1, url.length) : url,
```

Comment:
Could we not move this logic (of stripping a leading forward slash) into the validation store? Then we would only have to write it once, rather than duplicate this code fragment in a bunch of different places. Maybe there's a reason for it though, but it just looks a bit more verbose now that I though it needed to.

I notice you removed some of those checks from the validation store though in favour of adding them here so I presume there's a reason.

Reply:
I've done it this way so that this could be shared between the onboarding creation flow and the existing new app creation flow. Since the existing new app creation flow uses a leading slash and we don't want to allow the new onboarding to use that (presumably) this seemed like the best way. I don't think I removed anything else except what pertains to this, but I could be wrong.

Reply:
OK, I still don't think I understand the harm in putting it in the store since it's conditionally removing it and the validation should really be identical anyway since it's both just creating an app, so using the same endpoint and would expect the same format of URL - there shouldn't be any differences. But this one has been open long enough so I'm happy just to get it in since it solves the original issue.

Reply:
Unless I'm misunderstanding it, it's because if we conditionally allowed leading slashes in the store then the onboarding flow would also allow users to enter urls with leading slashes and still have them be valid. Which might go against the idea of the UX of the new flow 🤷 

Happy to come back to this, but I'll merge for now since you approved.